src/lib/layers/editable-geojson-layer.js

```diff
@@ -28,6 +29,8 @@ const defaultProps = {
   // Edit and interaction events
   onEdit: () => {},
 
+  hintPointMaxDistance: 0,
```

Comment:
All new props need documentation. But I'm OK with this being a constant for now.